portal/server/routes/contract/about/companiesHouseSearch.js

```diff
@@ -15,44 +16,15 @@ const getPortalCountryForCompaniesHouseCountry = (companiesHouseCountry) => {
 
 const router = express.Router();
 
-const getIndustryFromSicCode = (industrySectors, sicCodes) => {
-  let result = {};
-  if (!sicCodes) {
-    return null;
-  }
-
-  const sicCode = sicCodes[0];
-
-  industrySectors.forEach((sector) =>
-    sector.classes.find((industryClass) => {
-      if (industryClass.code === sicCode) {
-        result = {
-          sector: {
-            code: sector.code,
-            name: sector.name,
-          },
-          class: {
-            code: industryClass.code,
-            name: industryClass.name,
-          },
-        };
-        return result;
-      }
-      return null;
-    }),
-  );
-
-  return result;
-};
-
-router.post('/contract/:_id/about/supplier/companies-house-search/:prefix', provide([DEAL, INDUSTRY_SECTORS, COUNTRIES]), async (req, res) => {
+router.post('/contract/:_id/about/supplier/companies-house-search/:prefix', provide([DEAL, COUNTRIES]), async (req, res) => {
+  const { userToken } = requestParams(req);
```

Comment:
not sure i like this helper feels it's actually not doing much for us here ie
`  const { userToken } = requestParams(req)`
vs
`  const { userToken } = req.session`

-- Any reason we cant just do
`const { session: {userToken}, params: {prefix}, apiData: {deal}} =  req`?

Reply:
Good shout, changed.